borg/archiver.py

```diff
@@ -1174,12 +1173,12 @@ def parse_args(self, args=None):
             args = self.preprocess_args(args)
         parser = self.build_parser(args)
         args = parser.parse_args(args or ['-h'])
+        self.args = args
```

Comment:
why did you move this here and not keep all "self patching" in one place (see 1182)?